marklogic-data-hub-central/ui/src/components/entities/mapping/entity-settings/entity-settings.tsx

```diff
@@ -207,10 +206,10 @@ const EntitySettings: React.FC<Props> = (props) => {
                 <Col>
                   <Row>
                     <Col className={"d-flex"}>
-                      <Input
+                      <HCInput
                         id="targetPermissions"
                         placeholder="Please enter target permissions"
-                        data-testid={`${props.entityTitle}-targetPermissions`}
+                        dataTestid={`${props.entityTitle}-targetPermissions`}
```

Comment:
@lucianaabc-ml I see data-testid has been replaced with dataTestid, that might be the reason for a couple of RTL test failures. It should be **data-testid**